### ISSUE TITLE
Error: no Auth Provider found for name "oidc"

### ISSUE THREAD

**Issue by luisdavim** [KubeNetworks/kubevpn#45]
I'm getting the following error when trying to use this tool:

```
Error: no Auth Provider found for name "oidc"
```

adding the import:

```
_ "k8s.io/client-go/plugin/pkg/client/auth"
```

seems to solve the issue.

**Comment by wencaiwulue** [comment]
> 

Yes, you are right, i forgot it, client-go library already support, what we need is just refenerce this package, do you have interesting on this issue? maybe you can help me to solve this issue?

**Comment by wencaiwulue** [comment]
fixed in this commit: cc8eb9d93920fd26872ada8b126825813d5b757c

**Comment by luisdavim** [comment]
Thanks
